scheduler/src/cook/mesos.clj

```diff
@@ -189,7 +189,8 @@
    rebalancer-config        -- map, config for rebalancer. See scheduler/docs/rebalancer-config.adoc for details
    progress-config          -- map, config for progress publishing. See scheduler/docs/configuration.adoc for more details
    framework-id             -- str, the Mesos framework id from the cook settings
-   fenzo-config             -- map, config for fenzo, See scheduler/docs/configuration.adoc for more details"
+   fenzo-config             -- map, config for fenzo, See scheduler/docs/configuration.adoc for more details
+   sandbox-syncer-state     -- map, representing the sandbox syncer object"
```

Comment:
Should this be `sandbox-syncer` instead of `-state`?

Reply:
It is a map representing the atoms and functions that are returned as a result of starting the syncer. The map is not the sandbox syncer.